packages/core/src/indexing-store/postgres/store.ts

```diff
@@ -23,6 +27,7 @@ import {
 const MAX_BATCH_SIZE = 1_000 as const;
 const DEFAULT_LIMIT = 50 as const;
 const MAX_LIMIT = 1_000 as const;
+const LATEST = encodeCheckpoint(maxCheckpoint);
```

Comment:
Might just make sense to export `LATEST` from `utils/checkpoint.js` seeing how much it's used